docs-ui/stories/components/form-fields.stories.js

```diff
@@ -201,6 +202,14 @@ export const _RadioField = () => (
   </Form>
 );
 
+export const __FileField = () => (
+  <Form>
+    <FileField name="field" label="File Field" />
+  </Form>
+);
+
+__FileField.storyName = 'File';
```

Comment:
Why two `__` cc @vuluongj20 ?

Reply:
idk the file makes no sense there are weird patterns all over

Reply:
Oh yeah you're right lol